### DISCUSSION TITLE
How to add default workspace when reload window

### DISCUSSION THREAD

**Discussion by hiendvt** [eclipse-theia/theia#13950] [Q&A]
Hi,

I'm adding a default workspace when opening the ide for the first time. The way I do to open the default workspace is to extend the `DefaultWorkspaceServer` class and call `getDefaultWorkspace`. However, I close the workspace and can't reopen it automatically if I reload the window. Where can I check the flow code of the reload window?
![Screenshot from 2024-07-25 08-42-23](https://github.com/user-attachments/assets/216a36a3-b012-474a-9390-be6ed649a633)

Regards

**Comment by tsmaeder** [comment]
What do you mean, " I can't reopen it automatically"? If you reload the window, the behavior should be to reopen whatever workspace was loaded before. What's the scenario?
